samples/net/sockets/echo_client/overlay-802154.conf

```diff
@@ -15,4 +15,10 @@ CONFIG_NET_L2_IEEE802154=y
 CONFIG_NET_L2_IEEE802154_SHELL=y
 CONFIG_NET_L2_IEEE802154_LOG_LEVEL_INF=y
 
-CONFIG_NET_CONFIG_IEEE802154_CHANNEL=26
+#CONFIG_NET_CONFIG_IEEE802154_CHANNEL=26
+CONFIG_IEEE802154_CC13XX_CC26XX=n
+#CONFIG_NET_CONFIG_IEEE802154_DEV_NAME="IEEE802154_0"
+
+CONFIG_NET_CONFIG_IEEE802154_CHANNEL=1
+CONFIG_IEEE802154_CC13XX_CC26XX_SUB_GHZ=y
+CONFIG_NET_CONFIG_IEEE802154_DEV_NAME="IEEE802154_1"
```

Comment:
This does not look right. Perhaps provide separate overlay for sub ghz config instead. Same comment to changes in other samples.